pages/visualizations/chaoss/contributors_over_time.py

```diff
@@ -171,7 +170,9 @@ def create_graph(repolist, timer_pings, contribs, interval):
 
     # df for drive by contributros in time interval
     df_drive = (
-        df_drive_temp.groupby(by=df_drive_temp.created.dt.to_period(interval))["cntrb_id"]
+        df_drive_temp.groupby(by=df_drive_temp.created.dt.to_period(interval))[
```

Comment:
I find the first option more readable, especially if you have pandas experience.

Reply:
I agree, like the other change, the 'black' formatter did it and will continue to do it.

Reply:
let the war begin lol

Reply:
i think I can **technically** change the line-length requirement for 'black' but tbh I'd rather keep it as standard as possible. I'm a pep8 / let-the-formatter-do-what-it-wants person